CMakeLists.txt

```diff
@@ -107,6 +107,7 @@ option(NGRAPH_NOP_ENABLE "Control the building of the NOP backend" TRUE)
 option(NGRAPH_GPUH_ENABLE "Control the building of the Hybrid GPU backend" FALSE)
 option(NGRAPH_GENERIC_CPU_ENABLE "Enable build nGraph for generic CPU backend" FALSE)
 option(NGRAPH_DISTRIBUTED_ENABLE "Add distributed mode to the CPU backend" FALSE)
+option(NGRAPH_DISTRIBUTED_MLSL_ENABLE "Add distributed mlsl mode to the for CPU only backend" FALSE)
```

Comment:
This options is used like a sub option to NGRAPH_DISTRIBUTED_ENABLE. I think the option should only exist when NGRAPH_DISTRIBUTED_ENABLE is TRUE/ON.